andela/hermes-ah-backend
# 164798016 Feature password reset

#### What does this PR do?
This PR adds Password reset functionality via email
#### Description of Task to be completed?
- create routes requires
- create controllers for password reset
- validate input fields
- install npm packages
- add email notification
#### How should this be manually tested?
check out ft-password-reset-164798016
npm run start:dev
make a post request to api/v1/auth/reset with the email address on the body
retrieve the token from email
make a post request with new password to api/v1/auth/new_password
#### Any background context you want to provide?
I will raise another PR for the tests and documentation
#### What are the relevant pivotal tracker stories?
[164798016](https://www.pivotaltracker.com/n/projects/2318856/stories/164798016)
#### Screenshots

#### Questions:
